lib/peripheral.dart

```diff
@@ -33,27 +49,49 @@ class Peripheral {
       _manager.observePeripheralConnectionState(
           identifier, emitCurrentValue, completeOnDisconnect);
 
+  /// Check connection state of the Device.
   Future<bool> isConnected() => _manager.isPeripheralConnected(identifier);
 
+  /// Disconnects from Device if it's connected or cancels pending connection.
   Future<void> disconnectOrCancelConnection() =>
       _manager.disconnectOrCancelPeripheralConnection(identifier);
 
+  /// Discovers all Services, Characteristics and Descriptors for Device.
+  ///
+  /// Optional [transactionId] could be used to cancel operation.
   Future<void> discoverAllServicesAndCharacteristics({String transactionId}) =>
       _manager.discoverAllServicesAndCharacteristics(
           this, transactionId ?? TransactionIdGenerator.getNextId());
 
+  /// List of discovered Services for the Device.
   Future<List<Service>> services() => _manager.services(this);
 
+  /// List of discovered Characteristics for given Service.
+  ///
+  /// [servicedUuid] must be specified and characteristics only for that
+  /// service are returned.
   Future<List<Characteristic>> characteristics(String servicedUuid) =>
       _manager.characteristics(this, servicedUuid);
 
+  /// Reads RSSI for the device.
+  ///
+  /// Optional [transactionId] could be used to cancel operation.
   Future<int> rssi({String transactionId}) =>
       _manager.rssi(this, transactionId ?? TransactionIdGenerator.getNextId());
 
+  /// Request new MTU value for this device. This function currently is not
+  /// doing anything on iOS platform as MTU exchange is done automatically.
```

Comment:
Add info that if `requestMtu` was set during connection then it will also do nothing?

Reply:
It will actually result in an error :D